x-pack/plugins/apm/server/lib/backends/get_throughput_charts_for_backend.ts

```diff
@@ -81,7 +71,7 @@ export async function getThroughputChartsForBackend({
     response.aggregations?.timeseries.buckets.map((bucket) => {
       return {
         x: bucket.key + offsetInMs,
-        y: bucket.throughput.value,
+        y: calculateThroughput({ start, end, value: bucket.doc_count }),
```

Comment:
we should use the bucketSize here instead, but it uses `getMetricsDateHistogramParams` which sets the minimum bucket size to 60s. 

@smith / @dgieselaar can I remove the `getMetricsDateHistogramParams` and use the date_histogran with `fixed_interval` with the value returns by `getBucketSizeForAggregatedTransactions`?

Reply:
@cauemarcondes It looks like this is the same problem again where you are using the `start`/`end` instead of the bucket interval.